### ISSUE TITLE
Server Crash

### ISSUE THREAD

**Issue by tlacson7** [HerculesWS/Hercules#2717]
**Describe the bug**
(05/09/2020 14:36:03) [ Error ] : --- failed assertion --------------------------------------------
(05/09/2020 14:36:03) [ Error ] : socket.c:1320: 'fd >= 0' in function `wfifohead'
(05/09/2020 14:36:03) [ Error ] : 0x6322b6 nullpo_backtrace_print
(05/09/2020 14:36:03) [ Error ] : 	/home/rouser/hercules/src/common/nullpo.c:77
(05/09/2020 14:36:03) [ Error ] : 0x6322b6 assert_report
(05/09/2020 14:36:03) [ Error ] : 	/home/rouser/hercules/src/common/nullpo.c:147
(05/09/2020 14:36:03) [ Error ] : 0x4970b3 clif_send
(05/09/2020 14:36:03) [ Error ] : 	/home/rouser/hercules/src/map/clif.c:647
(05/09/2020 14:36:03) [ Error ] : 0x47c5e6 clif_guild_memberlogin_notice
(05/09/2020 14:36:03) [ Error ] : 	/home/rouser/hercules/src/map/clif.c:8028
(05/09/2020 14:36:03) [ Error ] : 0x4ae7af guild_recv_memberinfoshort
(05/09/2020 14:36:03) [ Error ] : 	/home/rouser/hercules/src/map/guild.c:1171
(05/09/2020 14:36:03) [ Error ] : 0x4bcfd6 intif_parse_GuildMemberInfoShort
(05/09/2020 14:36:03) [ Error ] : 	/home/rouser/hercules/src/map/intif.c:1217
(05/09/2020 14:36:03) [ Error ] : 0x4beb3c intif_parse
(05/09/2020 14:36:03) [ Error ] : 	/home/rouser/hercules/src/map/intif.c:2638
(05/09/2020 14:36:03) [ Error ] : 0x45a80e chrif_parse
(05/09/2020 14:36:03) [ Error ] : 	/home/rouser/hercules/src/map/chrif.c:1763
(05/09/2020 14:36:03) [ Error ] : 0x644c6d do_sockets
(05/09/2020 14:36:03) [ Error ] : 	/home/rouser/hercules/src/common/socket.c:1466
(05/09/2020 14:36:03) [ Error ] : 0x408eb1 main
(05/09/2020 14:36:03) [ Error ] : 	/home/rouser/hercules/src/common/core.c:539
(05/09/2020 14:36:03) [ Error ] : 0x7faf5d7c4504 ???
(05/09/2020 14:36:03) [ Error ] : 	???:0
(05/09/2020 14:36:03) [ Error ] : 0x409072 ???
(05/09/2020 14:36:03) [ Error ] : 	???:0
(05/09/2020 14:36:03) [ Error ] : 0xffffffffffffffff ???
(05/09/2020 14:36:03) [ Error ] : 	???:0
(05/09/2020 14:36:03) [ Error ] : --- end failed assertion ----------------------------------------
(05/09/2020 14:36:03) [ Error ] : Server received crash signal! Attempting to save all online characters!

Version: Latest
Pre-Renewal
20151029

**Comment by 4144** [comment]
using gepard?

**Comment by 4144** [comment]
or any other server modifications or plugins?

**Comment by tlacson7** [comment]
@4144  yes gepard

**Comment by 4144** [comment]
i can suspect gepard issue, because similar issue was at least on two servers with gepard.

crash also may happened unrelated to reported stack. try run map server with ``gdb`` and at next crash show output from gdb command ``bt``

**Comment by 4144** [comment]
can you show parts of code from each file reported in this stack?

**Comment by tlacson7** [comment]
Yes i will post. And also i enable gdb to get better error report incase it crash again
